jupyterhub/objects.py

```diff
@@ -136,9 +135,14 @@ class Hub(Server):
     of the server base_url.
     """
 
+    cookie_name = 'jupyter-hub-token'
```

Comment:
It should be `jupyterhub`, since it's the cookie for jupyterhub, but we can't change it without logging everybody out on upgrade. I've only moved it so that it's the class default.